en/identity-server/next/docs/references/conditional-auth/api-reference.md

```diff
@@ -657,6 +658,113 @@ This function returns the local user associated with the federate username given
       </tbody>
     </table>
 
+### HTTP GET
+
+`httpGet(url, headers)`
+
+The HTTP GET function enables sending HTTP GET requests to specified endpoints as part of the adaptive authentication scripts in WSO2 Identity Server. It's commonly used to interact with external systems or APIs to retrieve necessary data for authentication decisions.
+
+- **Parameters**
+
+    <table>
+      <tbody>
+        <tr>
+          <td><code>url</code></td>
+          <td>The URL of the endpoint to which the HTTP GET request should be sent.</td>
+        </tr>
+        <tr>
+          <td><code>headers</code></td>
+          <td>HTTP request headers to be included in the GET request (optional).</td>
+        </tr>
+      </tbody>
+    </table>
+
+  - **Example**
+
+  ``` js
+      function onLoginRequest(context) {
+          httpGet('https://example.com/resource', {
+              "Authorization": "Bearer token"
```

Comment:
shall we add another header and make it more clear on how to add multiple headers?
Or else add this point to the `headers` description

Reply:
+1. Updated the PR with the changes.